test/runtests.jl

```diff
@@ -341,4 +355,12 @@ end
     @test isempty(DataAPI.colmetadatakeys(tm))
 end
 
+@testset "rownumber" begin
+    table = TestTable([(a=1,), (a=2,)])
+
+    table[1] == (a=1,)
```

Comment:
what is this line needed for?

Reply:
This was just to make sure the TestTable is rightly implemented in case the implementation is modified in the future when *e.g.* using it in another example. I can remove it if it feels unnecessary.

Reply:
Then add `@test` in front of it. Currently this line is no-op essentially.

Reply:
Done!